ddtrace/profiling/recorder.py

```diff
@@ -20,7 +20,7 @@ def __missing__(self, key):
         raise KeyError(key)
 
 
-@attr.s(slots=True)
+@attr.s
```

Comment:
Why no longer using slotted class here?

Reply:
The `push_events` method gets read-only with `slots`, so it can't be replaced after fork.
Not using `slots` here should not be a big deal, there's only one `Recorder` per `Profiler` in the end 🤷